### ISSUE TITLE
Network Not Training, without any error

### ISSUE THREAD

**Issue by vyraun** [espnet/espnet#1281]
Hi,

My espnet set-up shows no issues in `make check_install`.

Yet, when I do `./run.sh` using the appropriate language numbers, the output just stops at 
`
stage 3: Network Training
`
and in the `train.log` file it's just:

```
# asr_train.py --config conf/train.yaml --ngpu 2 --backend pytorch --outdir exp/train_pytorch_train/results --tensorboard-dir tensorboard/train_pytorch_train --debugmode 1 --dict data/lang_1char/train_un$
# Started at Wed Oct 16 19:47:41 EDT 2019
#
2019-10-16 19:47:42,005 (asr_train:308) WARNING: Skip DEBUG/INFO messages
2019-10-16 19:47:42,009 (asr_train:326) WARNING: CUDA_VISIBLE_DEVICES is not set.


```

Even with `ngpu = 1` in `run.sh`, the same issue comes up. Any suggestions on how to fix this?

Thanks!

**Comment by gs5264** [comment]
you can try :source path.sh first.

**Comment by kan-bayashi** [comment]
Could you try following commands?
```bash
CUDA_VISIBLE_DEVICES=0 ./run.sh --stage3 --verbose 1
```

**Comment by vyraun** [comment]
> you can try :source path.sh first.

thanks @gs5264 @kan-bayashi , tried both source path.sh and the given command, same issue happens, after

`stage 3: Network Training`

there is no output:

```
# asr_train.py --config conf/train.yaml --ngpu 2 --backend pytorch --outdir exp/train_pytorch_train/results --tensorboard-dir tensorboard/train_pytorch_trai$
# Started at Fri Oct 18 16:19:39 EDT 2019
#
2019-10-18 16:19:40,122 (asr_train:308) WARNING: Skip DEBUG/INFO messages
```

**Comment by secretlyvogon** [comment]
Hi, I am having the same issue! I followed all the steps suggested and the training does not start. I get the following output

`# asr_train.py --config conf/train.yaml --preprocess-conf conf/specaug.yaml --ngpu 0 --backend pytorch --outdir exp/train_960_pytorch_train_specaug/results --tensorboard-dir tensorboard/train_960_pytorch_train_specaug --debugmode 1 --dict data/lang_char/train_960_unigram5000_units.txt --debugdir exp/train_960_pytorch_train_specaug --minibatches 0 --verbose 1 --resume --train-json dump/train_960/deltafalse/data_unigram5000.json --valid-json dump/dev/deltafalse/data_unigram5000.json 
# Started at Sun Dec  1 14:10:23 UTC 2019
#
/home/saket/espnet/tools/venv/lib/python3.7/site-packages/torch/nn/_reduction.py:49: UserWarning: size_average and reduce args will be deprecated, please use reduction='none' instead.
  warnings.warn(warning.format(ret))
2019-12-01 14:10:24,383 (asr_train:329) INFO: ngpu: 0
2019-12-01 14:10:24,383 (asr_train:332) INFO: python path = (None)
2019-12-01 14:10:24,383 (asr_train:335) INFO: random seed = 1
2019-12-01 14:10:24,386 (asr_train:352) INFO: backend = pytorch
2019-12-01 14:10:24,917 (deterministic_utils:24) INFO: torch type check is disabled`
